### PR TITLE
SYNR-1414 Fix synapser on Rscript

### PR DESCRIPTION
**Pull request by kimyen**

### CODE REVIEW

File: R/table.R [hunk 1]
```diff
@@ -12,7 +12,7 @@
     for (i in 1:dim(dataFrame)[2]) {
       if (is.numeric(dataFrame[[i]])) {
         dataFrame[[i]][is.nan(dataFrame[[i]])] <- "NaN"
-      } else if (is(dataFrame[[i]], "POSIXt")) {
+      } else if (method::is(dataFrame[[i]], "POSIXt")) {
```
Comment (karawoo) [inline R/table.R]:
```suggestion
      } else if (methods::is(dataFrame[[i]], "POSIXt")) {
```

File: R/table.R [hunk 2]
```diff
@@ -73,9 +73,9 @@
   if (synapseType=="BOOLEAN") {
     as.logical(list)
   } else if (synapseType == "DATE") {
-    if (is(list, "POSIXt")) {
+    if (method::is(list, "POSIXt")) {
```
Comment (karawoo) [inline R/table.R]:
```suggestion
    if (methods::is(list, "POSIXt")) {
```

File: R/table.R [hunk 3]
```diff
@@ -73,9 +73,9 @@
...
       .convertPOSIXToCharacterTimestamp(list)
-    } else if (is(list, "numeric")) {
+    } else if (method::is(list, "numeric")) {
```
Comment (karawoo) [inline R/table.R]:
```suggestion
    } else if (methods::is(list, "numeric")) {
```

**Comment by kimyen** [comment]
Thank you for checking @karawoo ! I will run the build before merging the code.

**Pull request merged**